src/partials/footer.html

```diff
@@ -65,6 +70,8 @@
             href="https://www.youtube.com/c/GoIT"
             class="footer-social-link"
             target="_blank"
+            rel="noopener noreferrer"
+            aria-label="YouTube link">
```

Comment:
Just remove extra > there and LGTM.

Reply:
![image](https://github.com/user-attachments/assets/8ce2ac94-b8e9-4344-a836-b02e8e077ff0)